src/deep-asset/package.json

```diff
@@ -49,7 +49,8 @@
     "url": "https://github.com/MitocGroup/deep-framework.git"
   },
   "scripts": {
-    "test": "babel-node `which isparta` cover _mocha -- --reporter spec --ui tdd",
+    "test": "babel-node `which isparta` cover --include 'lib/**/*.js' _mocha -- 'test/**/*.js' --reporter spec --ui tdd",
+    "debug-test": "`which istanbul` cover _mocha -- --compilers js:babel/register --reporter spec --ui tdd",
```

Comment:
@vcernomschi There's no reason to add "debug-test" command for each deep service, you can add it directly into test/bin/_head.sh file